core/domain/topic_fetchers.py

```diff
@@ -336,7 +386,9 @@ def get_multi_topic_summaries(topic_ids):
     return topic_summaries
 
 
-def get_published_topic_summaries():
+def get_published_topic_summaries() -> List[
+    Optional[topic_domain.TopicSummary]
+]:
```

Comment:
I think it is possible to remove `Optional` from here, because this function is only used in `contributor_dashboard.py` and there we are not handling the case of `None` https://github.com/oppia/oppia/blob/991261580a578df3145b8751f0f4255400ab3c57/core/controllers/contributor_dashboard.py#L615

Reply:
@vojtechjelinek @aks681 PTAL!

Reply:
Yep, lets do that.

Reply:
Done!